src/main/python/resources/annotaters/vep/template.py

```diff
@@ -0,0 +1,36 @@
+from abc import ABCMeta, abstractmethod
+import pandas as pd
+from src.main.python.resources.checkers.property_checker import PropertyChecker
+
+
+class Template(metaclass=ABCMeta):
+    def __init__(self, name, usable):
+        self.property_checker = PropertyChecker()
```

Comment:
> `PropertyChecker()` doesn't actually log (though pydoc claims `PropertyChecker.check_property()` does log). Was this chosen on purpose or was the goal to use `PropertyCheckerLogger()`?

Typo.

> Is there a reason to switch between these two on a **hardcoded** per-class basis instead of a single `PropertyChecker()` where the logging output is defined globally (and perhaps adjusting its logging level to better adhere to this "global logging level picture")?

Yes. When CAPICE could still produce logfiles itself, the order of initialized classes was very important (and still is). `CapiceManager()` is the first that should be initialized, as it provided the logger with the verbose flag(s) and the location of the logfile (which has since been disabled and removed in d1d95a643d65adcaa5ed449e79985f0ff04500e0 ). Since `CapiceManager()` also checks it's properties, it can't use the logger yet as it is not initialized yet. That's why there's both a `PropertyChecker()`, solely for `CapiceManager()` and a `PropertyCheckerLogger()` for remaining property checks.